examples/module-federation/app1/package.json

```
@@ -0,0 +1,28 @@
+{
+  "name": "app1",
+  "version": "1.0.0",
+  "main": "index.js",
+  "license": "MIT",
+  "scripts": {
+    "start": "webpack-cli serve",
+    "build": "webpack --mode production",
+    "serve": "serve dist -p 3001",
+    "clean": "rm -rf dist"
```

```suggestion
    "build": "webpack --mode production"
```
I think we don't need the latter two in this example? Also `rm -rf` is not cross platforms as far as I know, btw webpack has a built-in [`output.clean`](https://webpack.js.org/configuration/output/#outputclean) feature which should accommodate our needs?